torchao/float8/float8_scaling_utils.py

```diff
@@ -116,11 +128,13 @@ def _maybe_initialize_amaxes_scales_for_float8_cast(
         # Note: we need to enable distributed reduction here in order
         # to match numerics between single GPU and multi GPU code for
         # activations and gradients
-        new_amax = tensor_to_amax(x, reduce_amax=reduce_amax)
+
+        # We hardcode TensorWise Scaling on delayed scaling for now
```

Comment:
nit: throw an exception instead of silently changing the setting?

Reply:
Hmm so I actually I think for this PR:
`hp_tensor_to_float8_delayed` should actually assert that tile_size is None and then I can add the arg to this func with the same assert

Reply:
can also assert in `Float8LinearConfig.__post_init__`